### PR TITLE
Fix Redis suffix problem for snowflake

### PR DESCRIPTION
**Pull request by Yuqing-cat**
PR includes:
- create a common `get_online_test_table_name()` in `test_fixture`
- replace below with `online_test_table = get_online_test_table_name('nycTaxiCITable')`
```
now = datetime.now()
online_test_table = ''.join(['nycTaxiCITable', '_', str(now.minute), '_', str(now.second)])
```
- add table suffix logic in `snowflake_e2e`

### CODE REVIEW

**Review by xiaoyongzhu — DISMISSED** [review]

**Comment by xiaoyongzhu** [comment]
Calling get_online_test_table_name() in the same test from different function will yield different results, which needs to be addressed

**Comment by hangfei** [comment]
Release candidate for v0.4.0

File: feathr_project/test/test_fixture.py [hunk 1]
```diff
@@ -251,3 +252,8 @@ def add_new_dropoff_and_fare_amount_column(df: DataFrame):
     random.shuffle(derived_feature_list)
     client.build_features(anchor_list=[agg_anchor, request_anchor], derived_feature_list=derived_feature_list)
     return client
+
+def get_online_test_table_name(table_name: str):
+    # use different time for testing to avoid write conflicts
+    now = datetime.now()
+    return ''.join([table_name,'_', str(now.minute), '_', str(now.second)])
```
Comment (windoze) [inline feathr_project/test/test_fixture.py:259]: '_'.join([table_name, str(now.minute), str(now.second)])

Comment (Yuqing-cat) [reply feathr_project/test/test_fixture.py]: Agree. updated.

**Review by xiaoyongzhu — APPROVED** [review]

**Review by windoze — APPROVED** [review]

**Pull request merged**